sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AnalyzeWithPrebuiltModel.md

```diff
@@ -0,0 +1,114 @@
+# Analyze a document with a prebuilt model
+
+This sample demonstrates how to analyze data from certain types of common documents with prebuilt models, using an invoice as an example. For more information about prebuilt models and which types of documents are supported, see the [service documentation][docint_models].
+
+To get started you'll need a Cognitive Services resource or a Document Intelligence resource. See [README][README] for prerequisites and instructions.
+
+## Creating a `DocumentIntelligenceClient`
+
+To create a new `DocumentIntelligenceClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Document Intelligence API key credential by creating an `AzureKeyCredential` object that, if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateDocumentIntelligenceClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var client = new DocumentIntelligenceClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## Use a prebuilt model to analyze a document from a URI
+
+To analyze a given file at a URI, use the `AnalyzeDocument` method. The returned value is an `AnalyzeResult` object containing data about the submitted document. Since we're analyzing an invoice, we'll pass the model ID `prebuilt-invoice` to the method.
```

Comment:
nit: Here it refers to the `AnalyzeDocument` method, but the sample uses `AnalyzeDocumentAsync`. I wonder if it would be better to write all our samples using sync code. Do you know if the team has a preference?

Reply:
Can't say for the team as a whole but I've heard the opinion that our APIs are inherently async and that it's a good practice to handle them as async code. We had one single sync sample in the old Form Recognizer package but I decided to not include it here as I felt it didn't add much value.

For most samples I've been copying what we had in the old package. We also had mentioned methods without the `Async` suffix there, even when almost all samples used async APIs, and I think it's been working well.

I'll open an issue for now to see what other packages usually do and if we have a recommended approach here (https://github.com/Azure/azure-sdk-for-net/issues/40060).